lightningd/gossip_control.c

```diff
@@ -221,8 +221,73 @@ static const struct json_command getnodes_command = {
     "Returns a list of all nodes that we know about"};
 AUTODATA(json_command, &getnodes_command);
 
+static bool json_getroute_reply(struct subd *gossip, const u8 *reply, const int *fds,
+				struct command *cmd)
+{
+	struct json_result *response;
+	struct route_hop *hops;
+	size_t i;
+
+	fromwire_gossip_getroute_reply(reply, reply, NULL, &hops);
```

Comment:
Will do, I was wondering how to return an error, since all we can do is reply with the matching `*_reply` message type, and we probably do not want to fail `gossipd` because it got something unrecognizable from `lightningd` as part of a JSON-RPC request.